cli/src/pcluster/validators/cluster_validators.py

```diff
@@ -567,22 +588,22 @@ class EfsIdValidator(Validator):  # TODO add tests
     Validate if there are existing mount target in the head node availability zone
     """
 
-    def _validate(self, efs_id, head_node_avail_zone: str, are_all_security_groups_customized):
-        # Get head node availability zone
-        head_node_target_id = AWSApi.instance().efs.get_efs_mount_target_id(efs_id, head_node_avail_zone)
-        # If there is an existing mt in the az, need to check the inbound and outbound rules of the security groups
-        if head_node_target_id:
-            # Get list of security group IDs of the mount target
-            sg_ids = AWSApi.instance().efs.get_efs_mount_target_security_groups(head_node_target_id)
-            if not _check_in_out_access(sg_ids, port=2049, is_cidr_optional=are_all_security_groups_customized):
-                self._add_failure(
-                    "There is an existing Mount Target {0} in the Availability Zone {1} for EFS {2}, "
-                    "but it does not have a security group that allows inbound and outbound rules to support NFS. "
-                    "Please modify the Mount Target's security group, to allow traffic on port 2049.".format(
-                        head_node_target_id, head_node_avail_zone, efs_id
-                    ),
-                    FailureLevel.ERROR,
-                )
+    def _validate(self, efs_id, avail_zones: set, are_all_security_groups_customized):
```

Comment:
I know it's not part of your change but we still don't have unit tests for this validator.